docs/specs/clients/notify/data-structures.md

```diff
@@ -56,4 +56,21 @@
   "lastModified": Int64,
   "types": NotifyType[]
 }
-```
\ No newline at end of file
+```
+
+## Notify Server Subscriptions
+
+```jsonc
+NotifyServerSubscription[]
+```
+
+`NotifyServerSubscription`:
+```typescript
+{
+  topic: string, // dApp subscription topic to manage the subscription and call wc_notifySubscriptionUpdate and wc_notifySubscriptionDelete
+  account: Account, // CAIP-10 account
+  scopes: string[], // Array of notification types enabled for this subscription
+  sym_key: string, // Symetric key used for notification topic. sha256 to get notify topic
```

Comment:
```suggestion
  symKey: string, // Symetric key used for notification topic. sha256 to get notify topic
```
we use 🐫 in specs

Reply:
Fixed in 26ec86beb0576c667562e2023c3a4fe839728282